src/ziggurat/init.clj

```diff
@@ -115,7 +115,6 @@
 (defn stop-common-states []
   (mount/stop #'config/config
               #'metrics/statsd-reporter
-              #'rmqw/connection
```

Comment:
why have we romoved connection from common states?

Reply:
since there is no connection state for rabbitmq I remvoed it, I think in the same function I should call stop-connection